bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py

```diff
@@ -32,6 +34,7 @@
 
 for i, output_data in enumerate(outputs):
     print(f'OUTPUT #{i}')
+    # pylint: disable=consider-using-f-string
     print(
```

Comment:
Maybe instead of disabling lints, fix the issue?

Reply:
It this case, IMO, the `format` fits better than `f-string`. But, ok, I can change that

Reply:
You're right, my mistake. Perhaps we should disable this lint globally, it seems like a bad lint.

Reply:
brought this back

Reply:
Can we disable the lint globally?

Reply:
Oh, you did, so remove it here :)

Reply:
done